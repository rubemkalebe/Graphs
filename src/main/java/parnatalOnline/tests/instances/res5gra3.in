5 3
15 6 8 12
12 25 28
26 29
25
