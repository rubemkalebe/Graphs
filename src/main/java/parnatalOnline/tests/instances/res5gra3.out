38.000000
0 2, 0 3, 0 4, 1 2, 
120
0.002934;0.000000;